app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -147,6 +149,11 @@ class SearchFragment : Fragment() {
         super.onPrepareOptionsMenu(menu)
     }
 
+    private fun showSearchLayout(show: Boolean) {
+        rootView.searchLinearLayout.visibility = if (show) View.VISIBLE else View.GONE
+        rootView.fabSearch.visibility = if (show) View.VISIBLE else View.GONE
+    }
```

Comment:
Again, this is a very special case workaround, not a fix. This will only invite newer bugs in future.

You should fix one function, the core logic and not add special ifs and elses regarding this particular fragment